store/postgres/src/subgraphs.graphql

```diff
@@ -77,3 +78,17 @@ type EthereumContractEventHandler @entity {
     event: String!
     handler: String!
 }
+
+type EthereumContractDataSourceTemplate @entity {
+    id: ID!
+    kind: String!
+    name: String!
+    network: String
+    source: EthereumContractSourceTemplate!
+    mapping: EthereumContractMapping!
+}
+
+type EthereumContractSourceTemplate @entity {
```

Comment:
Shouldn't that be `EthereumContractDataSourceTemplateSource` to follow the same naming as the internal names ? :stuck_out_tongue_winking_eye: